modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/SelectEntityInput.es.js

```diff
@@ -106,6 +106,7 @@ class SelectEntityInput extends React.Component {
 							'criterion-input--error':
 								!value && renderEmptyValueErrors,
 						})}
+						data-testid="entity-select-displayed-input"
```

Comment:
I think that this testid is not being used.

Reply:
Done! thank you